loader-tests/export-opml-example.opml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not sure that we should put it in the `loader-tests` if we do not use it there.

Maybe to `core/test`? But I am open for arguments.